redbot/core/utils/antispam.py

```diff
@@ -45,14 +45,13 @@ def __interval_check(self, interval: AntiSpamInterval):
     @property
     def spammy(self):
         """
-        use this to check if any interval criteria are met
+        Check whether any intervals are active.
```

Comment:
It's a property so it probably makes more sense to refer to it as an attribute:
```suggestion
        Whether any intervals are active.
```
As said in other review comments, I feel it's not that clear what "intervals" means.